src/Localization/en/MaterializedViewsBuilder.json

```diff
@@ -44,5 +44,28 @@
     "ResizingDecisionText": "To understand if the Materializedviews Builder is the right size, ",
     "ResizingDecisionLink": "learn more about Materializedviews Builder sizing.",
     "WarningBannerOnUpdate": "Adding or modifying Materializedviews Builder instances may affect your bill.",
-    "WarningBannerOnDelete": "After deprovisioning the Materializedviews Builder, your materializedviews will not be updated with new source changes anymore. Materializedviews builder is compute in your account that performs read operations on source collection for any updates and applies them on materialized views as per the materializedview definition."
+    "WarningBannerOnDelete": "After deprovisioning the Materializedviews Builder, your materializedviews will not be updated with new source changes anymore. Materializedviews builder is compute in your account that performs read operations on source collection for any updates and applies them on materialized views as per the materializedview definition.",
+    "GlobalsecondaryindexesBuilderDescription": "Provision a Globalsecondaryindexes builder cluster for your Azure Cosmos DB account. Globalsecondaryindexes builder is compute in your account that performs read operations on source collection for any updates and updates the global secondary indexes as per the globalsecondaryindexes definition.",
```

Comment:
Can you change the format of every "Globalsecondaryindexes" reference to have proper spacing? It should be "Global secondary indexes" with a space between each word

Reply:
Updated